singlestoredb_cloud/metadata.csv

```diff
@@ -0,0 +1,22 @@
+metric_name,metric_type,interval,unit_name,per_unit_name,integration,description,orientation,short_name,curated_metric
+singlestoredb_cloud.cpu_usage_seconds_total,gauge,60,,,singlestoredb_cloud,Cumulative cpu time consumed in core-seconds,0,,
+singlestoredb_cloud.cpu_resource_limits,count,60,,,singlestoredb_cloud,The number of requested CPU resource limit,0,,
+singlestoredb_cloud.mem_used_b,count,60,byte,,singlestoredb_cloud,The memory used by all processes of SingleStoreDB Cloud,0,,
+singlestoredb_cloud.mem_total_b,count,60,byte,,singlestoredb_cloud,The memory limit for the SingleStoreDB Cloud node_id based on its cgroup membership in bytes.,0,,
+singlestoredb_cloud.disk_used_b,count,60,byte,,singlestoredb_cloud,The total bytes used by the disk,0,,
+singlestoredb_cloud.disk_total_b,count,60,byte,,singlestoredb_cloud,"The total size of the disk, in bytes",0,,
+singlestoredb_cloud.failed_write_queries,gauge,60,query,,singlestoredb_cloud,Number of write queries which have failed,0,,
+singlestoredb_cloud.successful_write_queries,gauge,60,query,,singlestoredb_cloud,Number of write queries which have successfully run,0,,
+singlestoredb_cloud.failed_read_queries,gauge,60,query,,singlestoredb_cloud,Number of read queries which have failed,0,,
+singlestoredb_cloud.successful_read_queries,gauge,60,query,,singlestoredb_cloud,Number of read queries which have successfully run,0,,
+singlestoredb_cloud.rows_affected_by_writes,gauge,60,row,,singlestoredb_cloud,Number of rows affected by write queries,0,,
+singlestoredb_cloud.rows_returned_by_reads,gauge,60,row,,singlestoredb_cloud,Number of rows returned by read queries,0,,
+singlestoredb_cloud.Execution_time_of_write,gauge,60,millisecond,,singlestoredb_cloud,Total wall-time of all writer queries since server startup.,0,,
```

Comment:
```suggestion
singlestoredb_cloud.execution_time_of_write,gauge,60,millisecond,,singlestoredb_cloud,Total wall-time of all write queries since server startup,0,,
```

Reply:
Could you please confirm that these updates are accurate?